quinn-proto/src/frame.rs

```diff
@@ -429,6 +430,19 @@ pub struct StreamMeta {
     pub fin: bool,
 }
 
+// This manual implementation exists because `Default` is not implemented for
+// `Range<u64>` on Rust 1.45. If the minimum version is bumped in the future,
+// this can be replaced with `derive(Default)`.
```

Comment:
Let's specifically use `MSRV` here so it's easier to find when we bump it.

```suggestion
// `Range<u64>` on Rust 1.45. If the MSRV is bumped in the future, this can be replaced
// with `derive(Default)`.
```